core/src/main/resources/application-dev.properties

```diff
@@ -13,7 +13,7 @@ spring.liquibase.enabled=true
 spring.liquibase.change-log=classpath:db/changelog/db.changelog-master.xml
 
 # Hibernate
-spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
Please revert. As far as I know in this project it is the responsibility of liquibase to update the database s hema.

Reply:
Okay done.